cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/etl/transform/ScriptFilterTransform.java

```diff
@@ -102,7 +102,7 @@ public void transform(StructuredRecord input, Emitter<StructuredRecord> emitter)
         metrics.count("filtered", 1);
       }
     } catch (Exception e) {
-      throw new IllegalArgumentException("Invalid filter condition.", e);
+      throw new IllegalArgumentException("Could not transform input: " + input, e);
```

Comment:
I think in general, the framework shouldn't log user data as it can easily become a security breach. For example, if I configure the script transform wrongly then suddenly all user sensitive data that is supposed to be readable by me is now readable by whoever can read system log (most likely operation people).